tonic_validate/metrics/retrieval_precision_metric.py

```diff
@@ -12,6 +12,13 @@
 class RetrievalPrecisionMetric(Metric):
     name: str = "retrieval_precision"
 
+    def __init__(self):
+        """
+        Metric that checks whether the context retrieved is relevant to answer the given question.
+        Returns a float between 0 and 1, where 1 is all the context is relevant and 0 is none of the context is relevant.
```

Comment:
Returns a float between 0 and 1. 1 indicates that all of the context is relevant. 0 indicates that none of the context is relevant.